docs/tutorials/preparing-an-application.md

```diff
@@ -0,0 +1,163 @@
+
+## Going From Docker-Compose to convox.yml
+
+If you are already using `docker-compose.yml` for your application moving to a `conovx.yml` is a relatively straightforward process. As an example take a look at the following `docker-compose.yml`
+
+```
+version: '2'
+services:
+  web:
+    image: httpd
+    ports:
+      - 80:80
+    volumes:
+      - /tmp/something
+    environment:
+      - MY_ENVIRONMENT: development
+    links:
+      - supportservice
+      - redis
+  web_secondary:
+    image: myusername/privateimage:latest
+    command: override_command.sh
+    ports:
+      - 3001
+    links:
+      - redis
+  redis:
+    image: "redis:alpine"
+  supportservice:
+    build:
+      context: .
+      dockerfile: Dockerfile.support
+      links:
+        - postgres
+  postgres:
+    image: postgres:12
+
+```
+
+So here we have 5 different services defined, one a httpd service, one a private image with an overridden command, one a plain Redis service, and one we're building locally from a custom Dockerfile, with a Postgres database image thrown in for good measure.
+
+The `convox.yml` manifest for this app would look like
+
+```
+resources:
+  redis:
+    type: redis
+  postgres:
+    type: postgres
+    options:
+      version: 12
+services:
+  web:
+    image: httpd
+    port: 80
+    volumes:
+      - /tmp/something
+    enviroment:
+      - MY_ENVIRONMENT=development
+    resources:
+      - redis
+  web_secondary:
+    image: myusername/privateimage:latest
+    command: override_command.sh
+    port: 3001
+    resources:
+      - redis
+  supportservice:
+    build:
+      path: .
+      manifest: Dockerfile.support
+    resources:
+      - postgres
+```
+
+Here you can see how we have turned the Redis and Postgres services into resources, which can be accessed by the other services. As we mentioned above by default those resources will be run in containers backed by durable storage but if you would prefer to use a cloud provider service you can do so using [resource overlays](https://docs.convox.com/reference/primitives/app/resource#overlays). We have also removed the links directive as services in a Rack are able to communicate each other using our built-in [service discovery](https://docs.convox.com/configuration/service-discovery). Otherwise, you will find the manifests to be nearly identical.
```

Comment:
```suggestion
Here you can see how we have turned the Redis and Postgres services into resources, which can be accessed by the other services. As we mentioned above, those resources will be run in containers backed by durable storage (by default) but if you would prefer to use a cloud provider service you can do so using [resource overlays](https://docs.convox.com/reference/primitives/app/resource#overlays). We have also removed the links directive as services in a Rack are able to communicate each other using our built-in [service discovery](https://docs.convox.com/configuration/service-discovery). Otherwise, you will find the manifests to be nearly identical.
```